adagrams/game.py

```diff
@@ -1,11 +1,130 @@
+import random
+
+SCORE_DICT = {
+    "AEIOULNRST": 1,
+    "DG": 2,
+    "BCMP": 3,
+    "FHVWY": 4,
+    "K": 5,
+    "JX": 8,
+    "QZ": 10,
+}
+
+
 def draw_letters():
-    pass
+    """Build a hand of 10 letters for the user
+
+    Parameters:
+    None
+
+    Returns:
+    list: letter_bank
+    """
+
+    letter_pool = {
+        "A": 9,
+        "B": 2,
+        "C": 2,
+        "D": 4,
+        "E": 12,
+        "F": 2,
+        "G": 3,
+        "H": 2,
+        "I": 9,
+        "J": 1,
+        "K": 1,
+        "L": 4,
+        "M": 2,
+        "N": 6,
+        "O": 8,
+        "P": 2,
+        "Q": 1,
+        "R": 6,
+        "S": 4,
+        "T": 6,
+        "U": 4,
+        "V": 2,
+        "W": 2,
+        "X": 1,
+        "Y": 2,
+        "Z": 1,
+    }
+    letter_bank = []
+    freq = 10
+    while freq > 0:
```

Comment:
Instead of having a variable that you have to update manually, you can do `while len(letter_bank) < 10`